packages/react-scripts/config/paths.js

```diff
@@ -132,6 +134,7 @@ if (
     proxySetup: resolveOwn(`${templatePath}/src/setupProxy.js`),
     appNodeModules: resolveOwn('node_modules'),
     swSrc: resolveModule(resolveOwn, `${templatePath}/src/service-worker`),
+    cachePath: resolveApp('node_modules/.cache'),
```

Comment:
We probably need to ensure that `node_modules` dir is writable. Since the`eslint-loader` (use in v3), it will fallback to write to `os.tempdir`. 

I don't know how `ESLint` handles the case when `cacheLocation` is non-writable.